### PR TITLE
docs(app): Add enum for app log level in json schema

### PR DESCRIPTION
**Pull request by hbiede**
### Description

Since only specified values are allowed, then suggest them as an enum

### Release Summary

Add enum for `app_log_level` in json schema

### Checklist

- I read the [Contributor Guide](../CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
  - [X] Yes
- My change supports the following platforms;
  - [X] `Android`
  - [X] `iOS`
- My change includes tests;
  - [ ] `e2e` tests added or updated in `packages/\*\*/e2e`
  - [ ] `jest` tests added or updated in `packages/\*\*/__tests__`
- [ ] I have updated TypeScript types that are affected by my change.
- This is a breaking change;
  - [ ] Yes
  - [X] No

### Test Plan

---

Think `react-native-firebase` is great? Please consider supporting the project with any of the below:

- 👉 Star this repo on GitHub ⭐️
- 👉 Follow [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter

:fire:

### CODE REVIEW

**Comment by hbiede** [comment]
@invertase @Salakar Is there something else I need to do here?

File: packages/app/firebase-schema.json [hunk 1]
```diff
@@ -43,7 +43,13 @@
         },
         "app_log_level": {
           "description": "Set the log level across all modules. Only applies to iOS currently. Can be 'error', 'warn', 'info', 'debug'.\n Logs messages at the configured level or lower.\n Note that if an app is running from AppStore, it will never log above info even if level is set to a higher (more verbose) setting",
-          "type": "string"
+          "type": "string",
+          "enum": [
+            "error",
+            "warm",
```
Comment (mikehardy) [inline packages/app/firebase-schema.json:49]: typo fix :-)

```suggestion
            "warn",
```

**Review by mikehardy — APPROVED** [review]
Looks good - I was unaware that an enum was possible - haven't played much with JSON schemas - and was out on summer vacation but I'm just getting back to things now. Thanks for your patience!

File: packages/app/firebase-schema.json (hunk 1 above)
Comment (hbiede) [reply packages/app/firebase-schema.json]: Whoops

**Pull request merged**